types/video.js/index.d.ts

```diff
@@ -138,7 +138,7 @@ declare namespace videojs {
      * @return    A player instance or `undefined` if there is no player instance
      *          matching the argument.
      */
-    function getPlayer(id: string): Player;
+    function getPlayer(id: string | Element): Player;
```

Comment:
it also is missing proper return type

Reply:
changed return type to `Player | undefined`